docs/resources/tanzu_kubernetes_cluster.md

```diff
@@ -7,9 +7,13 @@ Description: |-
 # Tanzu Kubernetes Cluster (Class-based Cluster) Resource
 
 This resource enables users to create and configure a TKG cluster for both TKGM & TKGS.
```

Comment:
In this doc - TKGM and TKGS are used. Those are internal names and one should refer to said products as: Tanzu Kubernetes Grid Service clusters running in vSphere for TKGm And Tanzu Kubernetes Grid Service for TKGs

Reply:
fixed.